examples/pi.cxx

```diff
@@ -6,20 +6,31 @@
 
 int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
 {
-  typedef RAJA::seq_reduce reduce_policy;
-  typedef RAJA::seq_exec execute_policy;
+  typedef RAJA::omp_target_reduce reduce_policy;
```

Comment:
There should be an entirely new unit test for target openmp instead of modifying the existing one